compiler/plugins/target/CUDA/CUDATarget.cpp

```diff
@@ -55,59 +55,61 @@ namespace mlir::iree_compiler::IREE::HAL {
 namespace {
 struct CUDAOptions {
   bool dumpPtx = false;
-  std::string clTargetChip = "sm_60";
-  std::string clTargetFeature = "+ptx76";
+  std::string clTarget = "sm_60";
+  std::string clTargetFeatures = "+ptx76";
   bool clUsePtxas = false;
   std::string clUsePtxasFrom;
   std::string clUsePtxasParams;
 
   void bindOptions(OptionsBinder &binder) {
     static llvm::cl::OptionCategory category("CUDA HAL Target");
-    binder.opt<bool>("iree-hal-cuda-dump-ptx", dumpPtx, llvm::cl::cat(category),
+    binder.opt<bool>("iree-cuda-dump-ptx", dumpPtx, llvm::cl::cat(category),
                      llvm::cl::desc("Dump ptx to the debug stream."));
 
-    binder.opt<std::string>("iree-hal-cuda-llvm-target-arch", clTargetChip,
-                            llvm::cl::cat(category),
-                            llvm::cl::desc("LLVM target chip."));
+    binder.opt<std::string>(
+        "iree-cuda-target", clTarget, llvm::cl::cat(category),
+        llvm::cl::desc("CUDA target as expected by LLVM NVPTX backend; e.g., "
```

Comment:
nit: drop the `,` comma? Seems like unnecessary punctuation.
```suggestion
        llvm::cl::desc("CUDA target as expected by LLVM NVPTX backend; e.g. "
```